pcsc-sys/src/lib.rs

```diff
@@ -41,7 +41,14 @@ pub type LONG = i32;
 #[cfg(target_os = "macos")]
 pub type ULONG = u32;
 
+#[cfg(target_os = "windows")]
+pub type SCARDCONTEXT = usize;
```

Comment:
`LONG_PTR` is signed, so this should be `isize` not `usize`.

Reply:
But it is `ULONG_PTR` actually :grin:

Reply:
Oh, dunno why I thought it was `LONG_PTR`.

OK then, I hope the different signs don't cause portability issues.